src/cfml/system/services/ServerService.cfc

```diff
@@ -526,6 +526,11 @@ component accessors="true" singleton {
 	// Serialize tray options and write to temp file
 	var trayOptionsPath = serverInfo.serverHome & '/trayOptions.json';
 	fileWrite( trayOptionsPath,  serializeJSON( serverInfo.trayOptions ) );
+
+
+	var startupTimeout = 120;
+	// Increase our startup allowance for Adobe engines, since a number of files are generated on the first request
+	if( CFEngineName == 'adobe' ) startupTimeout=240;
```

Comment:
Sure. Oops!  I didn't realize that was in the syntax guide...  I'll have to read it again. :)